src/lib.rs

```diff
@@ -83,9 +83,9 @@ for entry in WalkDir::new("foo").follow_links(true) {
 
 [`follow_links`]: struct.Walkdir.html#method.follow_links
 
-# Example: skip hidden files and directories efficiently on unix
+# Example: skip hidden files and directories on unix
 
-This uses the [`filter_entry`] iterator adapter to avoid yielding hidden files
+This uses the [`filter_entry`] iterator adapter in order to yield hidden files
```

Comment:
I think the previous wording is correct. To "yield" something is to provide it. We want to ignore hidden files, not yield them.